samples/nrf9160/modem_shell/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -0,0 +1,73 @@
+
+#Overlay to use nRF7002EK on top of nrf9160DK for Wi-Fi scanning
+
+# Disable Modem traces as uart1 disabled
+CONFIG_NRF_MODEM_LIB_TRACE=n
+
+# Need to disable some MoSh features to have more SRAM
+CONFIG_MOSH_WORKER_THREADS=n
+CONFIG_MOSH_IPERF3=n
+CONFIG_MOSH_CURL=n
+CONFIG_MOSH_SOCK=n
+CONFIG_MOSH_SMS=n
+CONFIG_MOSH_STARTUP_CMDS=n
+CONFIG_MOSH_AT_CMD_MODE=n
+CONFIG_MOSH_FOTA=n
+CONFIG_SHELL_CMD_BUFF_SIZE=512
+
+CONFIG_LOCATION_METHOD_WIFI=y
+CONFIG_LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD=y
+
+CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT=60
+CONFIG_LOCATION_WORKER_STACK_SIZE=8192
+
+# Does not work with buttons&LEDs
```

Comment:
>Is the LED issue with CONFIG_DK_LIBRARY resolved now?
@krish2718. No, I had to disable it with nRF7002EK. Otherwise, there are asserts:
`mosh:~$ ASSERTION FAIL [0] @ WEST_TOPDIR/nrf/lib/dk_buttons_and_leds/dk_buttons_and_leds.c:186
[00:00:04.665,710] <err> os: r0/a1:  0x00000004  r1/a2:  0x000000ba  r2/a3:  0x00000000
[00:00:04.665,740] <err> os: r3/a4:  0x00024275 r12/ip:  0x00000000 r14/lr:  0x00027095
[00:00:04.665,740] <err> os:  xpsr:  0x41000041
[00:00:04.665,771] <err> os: s[ 0]:  0x00000000  s[ 1]:  0x00000000  s[ 2]:  0x00000000  s[ 3]:  0x00000000
[00:00:04.665,802] <err> os: s[ 4]:  0x00000000  s[ 5]:  0x00000000  s[ 6]:  0x00000000  s[ 7]:  0x00000000
[00:00:04.665,832] <err> os: s[ 8]:  0x00000000  s[ 9]:  0x00000000  s[10]:  0x00000000  s[11]:  0x00000000
[00:00:04.665,832] <err> os: s[12]:  0x00000000  s[13]:  0x00000000  s[14]:  0x00000000  s[15]:  0x00000000
[00:00:04.665,863] <err> os: fpscr:  0x0005fdbc
[00:00:04.665,863] <err> os: r4/v1:  0x00000000  r5/v2:  0x20022438  r6/v3:  0x00000080
[00:00:04.665,893] <err> os: r7/v4:  0x0005f0b1  r8/v5:  0x0004b34c  r9/v6:  0x0006de14
[00:00:04.665,924] <err> os: r10/v7: 0x00000000  r11/v8: 0x00000000    psp:  0x20021698
[00:00:04.665,954] <err> os: EXC_RETURN: 0x0
[00:00:04.665,985] <err> os: Faulting instruction address (r15/pc): 0x000423a0
[00:00:04.666,015] <err> os: >>> ZEPHYR FATAL ERROR 4: Kernel panic on CPU 0
[00:00:04.666,015] <err> os: Fault during interrupt handling

[00:00:04.666,046] <err> os: Current thread: 0x2000cd68 (idle)
[00:00:04.845,550] <err> os: Halting system`

Reply:
For now, CONFIG_DK_LIBRARY is disabled with nRF7002.